server/api/chat/chat.controllers.js

```diff
@@ -1,6 +1,10 @@
 import Chat from "./chat.js";
 import { asyncHandler } from "../index.js";
 
+Array.prototype.sortNewestFirst = function () {
```

Comment:
remove from prototyle, define as a regular function.
plus, it should not be defined here, its not logical because its not a chat specific function. put in utils